book/01-introduction/sections/first-time-setup.asc

```diff
@@ -11,6 +11,9 @@ Die Konfiguration ist an drei verschiedenen Orten gespeichert:
 1. Die Datei `/etc/gitconfig`: enthält Werte, die für jeden Benutzer auf dem System und alle seine Repositorys gelten.
   Wenn Sie die Option `--system` an `git config` übergeben, liest und schreibt sie spezifisch aus dieser Datei.
   (Da es sich um eine Systemkonfigurationsdatei handelt, benötigen Sie Administrator- oder Superuser-Rechte, um Änderungen daran vorzunehmen.)
```

Comment:
Beziehst du dich auf diese Zeile?
Dann ist das "-" Zeichen ein normaler "Divis", wie er über die Tastatur über das Minuszeichen eingegeben wird.

Ich habe die Zeile mehrfach mit verschiedenen Zeilenbreiten und -umbrüchen getestet. Der "-" Strich hat sich nie wie ein "Soft Hyphen" verhalten.
Wie hast du das analysiert?

Reply:
Im Kommentar steht, dass der "Soft Hyphen" im Wort "persönlich" ist. Hier auf GitHub und vermutlich auch in deinem Editor wird er dir nicht angezeigt. In deinem Editor kannst du aber merken, dass es noch ein weiteres Zeichen gibt, wenn du Cursor zwischen "r" und "s" stellst und dann mit den Pfeiltasten nach rechts drückst. Zwischen "r" und "s" wirst du dann feststellen, dass du zweimal nach rechts drücken musst.

Du kannst ihn "sichtbar" machen, indem du "persönlich" in deine Zwischenablage kopierst und dann z.B. in Acrobat Reader bei einem beliebig geöffneten PDF die Suchen-Funktion mit <kbd>Strg</kbd>+<kbd>F</kbd>öffnest und in dieses Feld die Zwischenablage einfügst.

Im PDF (Seite 24 v2.1.87) kannst du ebenfalls "sehen", dass es den Soft Hyphen gibt, weil "persönlich" dort umbrochen ist.

Reply:
OK, raus damit.
Das war mit Sicherheit von mir nicht absichtlich eingegeben. Die Zeile könnte noch aus der uralten Übersetzung des ersten Buchs stammen.